codecov.yml

```diff
@@ -3,6 +3,7 @@ ignore:
   - "l2geth"
   - "**/*.t.sol"
   - "op-bindings/bindings/*.go"
+  - "packages/contracts-bedrock/contracts/(vendor|libraries)/(trie|rlp)"
```

Comment:
There are rlp tests in this repo?

Reply:
This code is tested in the contracts package, but not here, so it makes the numbers look lower unless excluded.

Reply:
Compare [this branch](https://app.codecov.io/gh/ethereum-optimism/optimism/tree/coverage%2Fbadge-and-ignore/packages/contracts-bedrock), to [develop](https://app.codecov.io/gh/ethereum-optimism/optimism/tree/develop/packages/contracts-bedrock).